src/style.css

```
@@ -0,0 +1,151 @@
+* {
+  box-sizing: border-box;
+  margin: 0;
+  font-family: Roboto, sans-serif;
+  padding: 0;
+}
+
+.card {
+  width: 200px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  padding: 32px 16px 16px;
+  background-color: #fff;
+
+  --min-element-margin: 16px;
+  --size-text-s: 10px;
+  --size-text-m: 12px;
+  --size-text-l: 14px;
+  --size-text-xl: 16px;
+  --color-gray: #616070;
+  --color-dark-blue: #060b35;
+  --color-blue: #00acdc;
+}
+
+.card__img-postion {
+  display: flex;
+  justify-content: center;
+  margin-bottom: 40px;
+}
+
+.card__img {
+  width: 160px;
+  height: 134px;
+}
+
+.card__title {
+  margin-bottom: 4px;
+
+  font-size: var(--size-text-m);
+  font-weight: 500;
+  line-height: 18px;
+  color: var(--color-dark-blue);
+}
+
+.card__id-text {
+  font-weight: 400;
+  font-size: var(--size-text-s);
+  line-height: 14px;
+  color: var(--color-gray);
+
+  margin-bottom: var(--min-element-margin);
+}
+
+.card__star-reviews-position {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+
+  margin-bottom: 24px;
+}
+
+.card__reviews {
+  font-weight: 400;
+  font-size: var(--size-text-s);
+  line-height: 14px;
+  color: var(--color-dark-blue);
+}
+
+.card__price-position {
+  display: flex;
+  justify-content: space-between;
+  margin-bottom: var(--min-element-margin);
+}
+
+.card__price-text {
+  font-weight: 400;
+  font-size: var(--size-text-m);
+  line-height: 18px;
+  color: var(--color-gray);
+}
+
+.card__price {
+  font-weight: 700;
+  font-size: var(--size-text-xl);
+  line-height: 18px;
+  color: var(--color-dark-blue);
+}
+
+.card__button-position {
+  display: flex;
+  justify-content: center;
+}
+
+.card__button {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+
+  font-style: bold;
+  font-weight: 700;
+  font-size: var(--size-text-l);
+  line-height: 16px;
+  text-transform: uppercase;
+  text-decoration: none;
+
+  cursor: pointer;
+
+  width: 166px;
+  height: 40px;
+
+  background-color: var(--color-blue);
+  border: 1px solid var(--color-blue);
+  border-radius: 5px;
+  color: #fff;
+}
+
+.card__button:hover {
+  background-color: #fff;
+  color: var(--color-blue);
+}
+
+.stars__star {
+  list-style: none;
+  height: 16px;
+  width: 16px;
+  background-image: url(./images/star.svg);
+  background-repeat: no-repeat;
+  margin-right: 4px;
+  background-position: center;
+}
+
+.stars__star:last-child {
+  margin-right: 0;
+}
+
+.block__stars {
+  display: flex;
+  flex-direction: column;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars--1 li:first-child,
+.stars--2 li:nth-child(-n + 2),
+.stars--3 li:nth-child(-n + 3),
+.stars--4 li:nth-child(-n + 4),
+.stars--5 li:nth-child(n) {
+  background-image: url(./images/star-active.svg);
```

it's better not to use tag. please add class 
```suggestion
.stars--1 .star:first-child,
.stars--2 .star:nth-child(-n + 2),
.stars--3 .star:nth-child(-n + 3),
.stars--4 .star:nth-child(-n + 4),
.stars--5 .star:nth-child(n) {
  background-image: url(./images/star-active.svg);
```